### PR TITLE
[pipeline-connector][doris] add doris pipeline connector.

### PR DESCRIPTION
**Pull request by lvyanquan**
Authered and committed by @JNSimba, copy from https://github.com/ververica/flink-cdc-connectors/pull/2729 to rebase master and squash commit.

### CODE REVIEW

**Comment by lvyanquan** [comment]
@JNSimba @leonardBang  CC.

File: flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-doris/pom.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            <version>${revision}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-runtime</artifactId>
+            <version>${flink.version}</version>
+            <scope>provided</scope>
+        </dependency>
```
Comment (JNSimba) [inline flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-doris/pom.xml:52]: Some of dependency   already exist in the parent, and additional imports are needed here ?

Comment (lvyanquan) [reply flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-doris/pom.xml]: done.

**Review by JNSimba — APPROVED** [review]
LGTM

**Review by PatrickRen — APPROVED** [review]
@lvyanquan @JNSimba Thanks for the contribution! I checked the packaged JAR and found nothing suspicious. +1

**Pull request merged**